aten/src/ATen/native/cuda/UnaryComplexKernels.cu

```diff
@@ -20,7 +24,7 @@ __host__ __device__ static inline c10::complex<T> angle_wrapper(c10::complex<T>
 }
 
 void angle_kernel_cuda(TensorIterator& iter) {
-  AT_DISPATCH_ALL_TYPES_AND_COMPLEX(iter.dtype(), "angle_cuda", [&]() {
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES(iter.common_dtype(), "angle_cuda", [&]() {
```

Comment:
Note the dispatch is on the common_dtype here

Reply:
@kshitij12345 @mruberry why did we change it to `common_dtype`?

Reply:
Reference: 

https://github.com/pytorch/pytorch/pull/45733#discussion_r504247931